README.md

```diff
@@ -12,6 +12,7 @@
   methods in GitHub).
 - Go to the Migration Kit folder: `spacelift-migration-kit`.
 - Install the Python dependencies and the `spacemk` command in a Python virtual environment: `poetry install`.
+  - To install poetry: `pip install poetry`.
```

Comment:
Poetry is listed as a prerequisite in the previous section and I would rather not suggest an installation method because people might have different preferences and constraints.